packages/eslint-plugin/src/rules/no-empty-styled-expression/index.ts

```diff
@@ -25,9 +25,20 @@ const createNoEmptyStyledExpressionRule =
     return {
       'CallExpression[callee.type="MemberExpression"]': (node: CallExpression) => {
         const { references } = context.getScope();
-        const isStyledImported = references.some((reference) =>
-          reference.resolved?.defs.some(isStyledImportSpecifier)
-        );
+
+        if (node.callee.type !== 'MemberExpression') {
```

Comment:
Is this needed? Shouldn't `CallExpression[callee.type="MemberExpression"]` already filter on that?

Reply:
I also thought this, but the code is giving type errors when I don't check for the node.callee.type either implicitly or explicitly. I will try to find an alternative if I can!

Reply:
Oh I see, it's for type checking. Probably not much you can do, since the types don't accept generic arguments.